src/main/java/seedu/address/model/person/Phone.java

```diff
@@ -15,19 +18,47 @@
             "Phone numbers can only contain numbers, and should be at least 3 digits long";
     public static final String PHONE_VALIDATION_REGEX = "\\d{3,}";
     public final String value;
+    public final ArrayList<String> phonelist;
```

Comment:
I recommend using` List<String>` over `ArrayList<String>` due to the advantageous reasons stated here:
https://stackoverflow.com/questions/2279030/type-list-vs-type-arraylist-in-java